### PR TITLE
Fix keyword format for downstream docs

### PR DESCRIPTION
**Pull request by mistyhacks**
Fixes https://github.com/docker/docker.github.io/issues/2687

baby iguana

### CODE REVIEW

File: docs/spec/api.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "HTTP API V2"
 description: "Specification for the Registry API."
-keywords: ["registry, on-prem, images, tags, repository, distribution, api, advanced"]
```
Comment (stevvooe) [inline docs/spec/api.md]: Make sure to hit the template, as well.

Comment (mstanleyjones) [reply docs/spec/api.md]: Good catch, done.

**Comment by stevvooe** [comment]
@mstanleyjones Are you sure these aren't supposed to be `["registry", "foo", "bar", ...]`?

**Comment by mstanleyjones** [comment]
99.9% sure. @johndmulhausen PTAL

**Comment by stevvooe** [comment]
LGTM

**Comment by mistyhacks** [comment]
I'm not authorized to merge. :) Let me open the cherry-pick PR too.

**Pull request merged**